docs/release-notes/docker-escape-hatch.rst

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  Docker: Introduces the configuration option, ``environment.docker_flags``, for experiments and jobs to allow
+   users to specify additional Docker configuration. Arguments and flags that ``docker run`` accepts
+   (that Determined does not set) can now be provided.
```

Comment:
:orphan:

**New Features**

-  Docker: Introduce a configuration option, ``environment.docker_flags``, for experiments and jobs, allowing
   users to specify additional Docker configuration. Arguments and flags accepted by ``docker run``
   (that are not already set by Determined) can now be provided.